proposals/0017-conforming-literals.md

```diff
@@ -187,6 +213,24 @@ The testing also revealed that the lack of complete overload sets for
 `asfloat16` is a barrier for users. Further discussion is needed to determine
 how best to address this problem.
 
+### Mitigating Migration Pain
+
+There are no observed instances of the change in floating point behavior causing
+disruption. If such cases exist the existing `-Wconversion` diagnostics will
+sufficiently notify users of implicit conversions that result in precision loss.

```

Comment:
Supporting a rewriter for this is going to be a huge effort, and observationally we don't have any indication that this would be a widespread problem. Are there cases where you don't think a `-Wconversion` warning would fire?

Curious for @damyanp's thoughts on the investment for a rewriter, but I feel like that makes this feature 10-20x larger.

Reply:
Could a rewriter be tracked through a separate proposal? I'd like more info on scenarios and where it'd be implemented but it seems that it doesn't have to block this proposal.

Reply:
Filed #205 to consider rewriter tooling.